mne/time_frequency/tfr.py

```diff
@@ -1121,7 +1120,7 @@ def tfr_multitaper(
 
     Notes
     -----
-    %(temporal-window_tfr_notes)s
+    %(temporal-window_tfr_multitaper_notes)s
```

Comment:
```suggestion
    %(temporal-window_tfr_multitaper_notes)s

```
here also we need an extra blank line to separate paragraphs